libcontainer/system/userns_deprecated.go

```diff
@@ -0,0 +1,17 @@
+package system
+
+import "github.com/opencontainers/runc/libcontainer/userns"
+
+var (
+	// RunningInUserNS detects whether we are currently running in a user namespace.
+	// Originally copied from github.com/lxc/lxd/shared/util.go
+	// Deprecated: use github.com/opencontainers/runc/libcontainer/userns.RunningInUserNS
```

Comment:
We haven't been guaranteeing Go compatibility of libcontainer, so I think we can just drop these deprecated functions, but not a strong opinion.

Reply:
Perhaps keep only "RunninginUserNS" (the "popular" one) and drop the others